djstripe/models.py

```diff
@@ -125,7 +126,9 @@ class Customer(StripeCustomer):
 
     # account = models.ForeignKey(Account, related_name="customers")
 
-    default_source = models.ForeignKey(StripeSource, null=True, related_name="customers")
+    default_source = models.ForeignKey(
```

Comment:
one line, but good catch on the ``on_delete``! Do you have a test for this case?